docs/configuration/v1.x/metrics/azure-sql-database.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Azure SQL Database Declaration
+---
+
+## Azure SQL Database - ![Availability Badge](https://img.shields.io/badge/Available%20Starting-v1.1.0-green.svg)
+
+You can scrape an Azure SQL Database via the `AzureSqlDatabase` resource type.
+
+The following fields need to be provided:
+
+- `serverName` - The name of the SQL Server instance.
+- `databaseName` - The name of the database.
+
+All supported metrics are documented in the official [Azure Monitor documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported#microsoftsqlserversdatabases).
+
+The following scraper-specific metric labels will be added:
+
+- `server` - The name of the SQL Server instance.
```

Comment:
I was looking at this again and was wondering, would it make sense if this was `serverName` instead @adamconnelly ?

Reply:
Yeah, I think that makes perfect sense.